packages/opentelemetry-api/.eslintignore

```diff
@@ -0,0 +1,2 @@
+build
```

Comment:
`build` and `build/` is the same?

Reply:
Maybe, we should include `**/node_modules` directory here?

Reply:
yes build and build/ is the same.
We could add `node_modules` but it's ignored by default.